draft-ietf-quic-transport.md

```diff
@@ -3605,12 +3601,12 @@ subsequent to the first do not need to fit within a single UDP datagram.
 
 #### Abandoning Initial Packets {#discard-initial}
 
-Endpoints cease both sending and processing Initial packets when it receives a
-Handshake packet containing a KEYS_READY frame.  Though packets might still be
-in flight or awaiting acknowledgment, no further Initial packets need to be
-exchanged beyond this point.  Initial packet protection keys are discarded (see
-Section 4.10 of {{QUIC-TLS}}) along with any loss recovery and congestion
-control state (see Sections 5.3.1.2 and 6.9 of {{QUIC-RECOVERY}}).
+Endpoints cease both sending and processing Initial packets when it both sends
+and receives a Handshake packet containing a KEYS_ACTIVE frame.  Though packets
```

Comment:
I think we should say "sends and receives a Handshake packet _or a 1-RTT packet_ containing a KEYS_ACTIVE frame", assuming that we wouldn't be waiting for ACKs when dropping the keys that are to carry the ACKs.

This is because a Handshake packet carrying just the KEYS_ACTIVE frame can be dropped, while all other packets reach the peers.